README.md

```diff
@@ -26,25 +27,35 @@ Following BSP requests are supported in the current implementation:
 - `workspace/buildTargets`
 - `workspace/reload`
 
-
 ## Architecture
 
 See [ARCHITECTURE.md](./ARCHITECTURE.md)
 
 ## Launch the Build Server for Gradle
+
 ### Specify the Plugin Location
-The main class of the build server is `com.microsoft.java.bs.core.Launcher`. When you launch the server, you need to specify the location of the Gradle plugin via the system property `plugin.dir`. By default, you will find the plugin jar at `server/build/libs/plugins/plugin.jar` after building the project.
+
+The main class of the build server is `com.microsoft.java.bs.core.Launcher`. When you launch the server, you need to
```

Comment:
I think those line breaks are not necessary. They do not affect the render result. If user is reading the raw markdown content, he can simply turn on word wrap

Reply:
Okay, it was actually done by my IDEs formatter.